app/src/main/java/uk/ac/ebi/atlas/monitoring/JsonBuildVersionController.java

```diff
@@ -9,28 +9,28 @@
 import org.springframework.web.bind.annotation.RestController;
 import uk.ac.ebi.atlas.controllers.JsonExceptionHandlingController;
 
-import static uk.ac.ebi.atlas.utils.GsonProvider.GSON;
+import java.util.Map;
 
 @RestController
-@PropertySource("classpath:configuration.properties")
+@PropertySource("classpath:resources/git.properties")
 public class JsonBuildVersionController extends JsonExceptionHandlingController {
     private final ImmutableMap<String, String> buildVersion;
 
-    public JsonBuildVersionController(@Value("${build.number}") String buildNumber,
-                                      @Value("${build.branch}") String buildBranch,
-                                      @Value("${build.commitId}") String buildCommitId,
-                                      @Value("${build.tomcatHostname}") String tomcatHostname) {
-        buildVersion = ImmutableMap.of(
-                "bambooBuildVersion", buildNumber,
-                "gitBranch", buildBranch,
-                "gitCommitID", buildCommitId,
-                "tomcatHostname", tomcatHostname);
+    public JsonBuildVersionController(@Value("${git.build.version}") String buildVersion,
+                                      @Value("${git.branch}") String gitBranch,
+                                      @Value("${git.commit.id}") String gitCommitId,
+                                      @Value("${git.commit.message.full}") String commitMessage) {
```

Comment:
what could be the reason to add a 'git.commit.message.full' instead of 'git. commit. message'?

Reply:
There is no `git.commit.message` property from this plugin.

It has these one:

- git.branch
- git.build.host
- git.build.user.email
- git.build.user.name
- git.build.version
- git.closest.tag.commit.count
- git.closest.tag.name
- git.commit.id
- git.commit.id.abbrev
- git.commit.id.describe
- git.commit.message.full
- git.commit.message.short
- git.commit.time
- git.commit.user.email
- git.commit.user.name
- git.dirty
- git.remote.origin.url
- git.tags
- git.total.commit.count

I prefer to use the `git.commit.message.full` than the `git.commit.message.short` as we have enough space on that page.